content/Contribute.tid

```diff
@@ -1,13 +1,12 @@
 created: 20110211110600000
 creator: psd
-modified: 20110211131000000
-modifier: blaine
+modified: 20130118141000000
+modifier: mkerrigan
 title: Contribute
 type: text/x-tiddlywiki
 
-~TiddlyWiki has an enthusiastic and friendly [[Community|Help and Support]] of people around the world helping to grow and improve it. But there's always more to do and we welcome any offers of assistance. There are many ways that you can help:
-* Testing and [[reporting bugs|https://github.com/TiddlyWiki/TiddlyWiki/issues]] against the core code. Clear, easily reproducible bug reports are incredibly useful and really help the team improve the quality of ~TiddlyWiki
-* [[Contributing code|http://trac.tiddlywiki.org/wiki/ContributingCode]]
-* [[Making translations|https://github.com/TiddlyWiki/translations]]
-* Documentation needs planning, writing and editing, particularly user guide information at http://www.tiddlywiki.org
-If you would like to make a financial contribution, ~TiddlyWiki is owned by the not-for-profit [[UnaMesa Foundation|http://www.unamesa.org/]] which welcomes donations.
\ No newline at end of file
+~TiddlyWiki has an enthusiastic and friendly [[Community]] of people around the world helping to grow and improve it. But there's always more to do and we welcome any offers of assistance. There are many ways that you can help:
+* Testing and [[reporting bugs|https://github.com/TiddlyWiki/TiddlyWiki/issues]] against the core code. Clear, easily reproducible bug reports are incredibly useful and help improve the quality of ~TiddlyWiki
+* [[Contributing code|https://github.com/TiddlyWiki/TiddlyWiki]]
+* Making [[translations|Translations]]
+* [[Documentation]]
```

Comment:
Let's make this line

    * Improving [[documentation|Documentation]]

(also that's a nice opportunity for you to practice creating a second commit in the same branch; note that since you have already created a PR but it is not merged yet, the second commit will be added to this PR automatically)